plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -25,6 +26,7 @@ function getHealthStatus(serviceID) {
   return (
     <div className="service-page-header-status page-header-breadcrumb-content-secondary muted">
       {`${serviceStatus} (${runningTasksCount} of ${instancesCount})`}
+      <ServiceStatusWarning item={service} showDebugInstruction={true} />
```

Comment:
I'm not sure I like an inversion of control here.
Do you think it's better to have two of them one with Debug instructions and one without?